dist/nix/templates/simple-neovim/flake.nix

```diff
@@ -0,0 +1,27 @@
+{
+  #change url when merged
+  inputs.quick-lint-js.url = github:countoren/quick-lint-js;
+  inputs.flake-utils.url = github:numtide/flake-utils;
+
+  outputs = { self, nixpkgs, flake-utils, quick-lint-js }:
+  flake-utils.lib.eachDefaultSystem (system:
+  let pkgs = nixpkgs.legacyPackages.${system};
+  in rec {
+    packages.default = pkgs.neovim.override {
+      configure = {
+        customRC = ''
+          let $PATH = $PATH.":${quick-lint-js.packages.${system}.quick-lint-js}/bin"
+        '';
+        packages.myPackages = {
+          start = with pkgs.vimPlugins;[
+            # loaded on launch
+              ale
+              quick-lint-js.packages.${system}.vimPlugin 
+            ];
+            # manually loadable by calling `:packadd $plugin-name`
+            opt = [ ];
+        };
+      };
+    };
```

Comment:
Must fix: Now that I learned more about Nix flakes, I think this makes sense. However, I think this belongs inside `/flake.nix`, not in a template. Please move this code into `/flake.nix`.

Reply:
Why nix flakes? as a package?
this will make it harder to modified by the user, we want to declare that this is meant to user to modify.
now we might want to create some more templates for people that has vim already and we just create them a file with additional customRC and the plugins adding a comment how they can import it. to their vim configurations.

Reply:
> as a package?

Yes. That way I can manually test quick-lint-js's Vim plugin with `nix run .#quick-lint-js-vim-ale` or `nix run .#quick-lint-js-neovim-lspconfig`. (I thought that was the point of adding a pre-configured Vim.)

> this will make it harder to modified by the user, we want to declare that this is meant to user to modify.

I think plain documentation is a better solution to this problem. (With your approach, we would need to document how to use the flake template *and* document how to modify the template.)

> now we might want to create some more templates for people that has vim already and we just create them a file with additional customRC and the plugins adding a comment how they can import it. to their vim configurations.

I thought that was what this template was. You are saying we would need *more* templates? That doesn't seem like a good plan.

Reply:
Ok maybe you have a point, I moved the template to packages,
I was able to build and run simple-neovim but you might to want confirm yourself it is fully working.
I think the template one was fully working.